cmd/export.go

```diff
@@ -270,3 +303,18 @@ structure:
 # or a URL that leads to the raw data
 # dataPath:
 `
+
+// FilenameToYAML takes a filename, removes any format ending (if there is one), and replaces it with yaml
+// should probably be generalized to
+// FilenameFormatChange(filename string, newEnding string) string
+func FilenameToYAML(filename string) string {
```

Comment:
I think instead of this custom function we could use a `filepath` one-liner whenever you need to manually construct a filename with extension:
```golang
fmt.Sprintf("%s.yaml", filepath.Base(filename))
```

there's also:
```golang
// dataset data formats have a string that matches their extension name:
fmt.Sprintf("%s.%s", filepath.Base(filename), dataset.JSONDataFormat.String())

// if you have a dataset with a structure, you can do this:
fmt.Sprintf("%s.%s", filepath.Base(filename), ds.Structure.Format.String())
```

This carries the added safety of working a little better cross-platform where path strings can get weird.